src/ompl/tools/thunder/src/SPARSdb.cpp

```diff
@@ -1278,13 +1378,14 @@ void ompl::geometric::SPARSdb::resetFailures()
 }
 
 void ompl::geometric::SPARSdb::findGraphNeighbors(base::State *st, std::vector<Vertex> &graphNeighborhood,
-                                                  std::vector<Vertex> &visibleNeighborhood)
+                                                  std::vector<Vertex> &visibleNeighborhood, double radius)
 {
     visibleNeighborhood.clear();
     stateProperty_[queryVertex_] = st;
-    nn_->nearestR(queryVertex_, sparseDelta_, graphNeighborhood);
+    if (radius < 0) radius = sparseDelta_;
```

Comment:
is `radius < 0` acceptable?

Reply:
This makes sure that it is not, it also sets the default radius to spardeDelta_
I did this since I am not able to set the default to sparse delta in the function declaration.